### PR TITLE
Compile bootstrap less in addition to ckan less

### PR DESCRIPTION
**Pull request by JackMc**

### CODE REVIEW

File: datacats/scripts/compile_less.sh [hunk 1]
```diff
@@ -1,2 +1,9 @@
 #!/bin/bash
-lessc $1 > $2
+
+
+lessc /project/target/ckan/ckan/public/base/less/main.less \
+> /project/target/ckan/ckan/public/base/css/main.debug.css
```
Comment (wardi) [inline datacats/scripts/compile_less.sh]: should we be generating the main.css too? I think when debug is off that's the css file that's actually used

Comment (JackMc) [inline datacats/scripts/compile_less.sh]: Yes, but this is mainly a debug feature right? We should do less compiling for prod on the server side, no?

Comment (wardi) [inline datacats/scripts/compile_less.sh]: ok, so should we fix this with documentation? Tell people if they want their style changes visible on production they need to copy them to the right place? That seems like an easy way to end up with grumpy users after they run 'deploy'

Comment (JackMc) [inline datacats/scripts/compile_less.sh]: Huh, well we could just do a copy, but that would imply that the non-minified sources are good for prod :stuck_out_tongue:

**Pull request merged**